Don't override the environment settings.  Also, don't try to set the local time
zone on the build system, and don't run ranlib twice.

--- Makefile
+++ Makefile
@@ -223,12 +223,12 @@ GCC_DEBUG_FLAGS = -Dlint -g3 -O3 -fno-co
 # before the first Monday in January when a "%V" format is used and January 1
 # falls on a Friday, Saturday, or Sunday.
 
-CFLAGS=
+CFLAGS ?=
 
 # Linker flags.  Default to $(LFLAGS) for backwards compatibility
 # to tzcode2012h and earlier.
 
-LDFLAGS=	$(LFLAGS)
+LDFLAGS ?=	$(LFLAGS)
 
 zic=		./zic
 ZIC=		$(zic) $(ZFLAGS)
@@ -289,13 +289,13 @@ GZIPFLAGS=	-9n
 
 ###############################################################################
 
-cc=		cc
+cc :=		$(CC)
 CC=		$(cc) -DTZDIR=\"$(TZDIR)\"
 
-AR=		ar
+AR ?=		ar
 
 # ':' on typical hosts; 'ranlib' on the ancient hosts that still need ranlib.
-RANLIB=		:
+RANLIB ?=	:
 
 TZCOBJS=	zic.o localtime.o asctime.o scheck.o ialloc.o
 TZDOBJS=	zdump.o localtime.o ialloc.o asctime.o
@@ -345,11 +345,10 @@ install:	all $(DATA) $(REDO) $(MANS)
 			$(DESTDIR)$(MANDIR)/man3 $(DESTDIR)$(MANDIR)/man5 \
 			$(DESTDIR)$(MANDIR)/man8
 		$(ZIC) -y $(YEARISTYPE) \
-			-d $(DESTDIR)$(TZDIR) -l $(LOCALTIME) -p $(POSIXRULES)
+			-d $(DESTDIR)$(TZDIR) -p $(POSIXRULES)
 		cp -f iso3166.tab zone.tab $(DESTDIR)$(TZDIR)/.
 		cp tzselect zic zdump $(DESTDIR)$(ETCDIR)/.
 		cp libtz.a $(DESTDIR)$(LIBDIR)/.
-		$(RANLIB) $(DESTDIR)$(LIBDIR)/libtz.a
 		cp -f newctime.3 newtzset.3 $(DESTDIR)$(MANDIR)/man3/.
 		cp -f tzfile.5 $(DESTDIR)$(MANDIR)/man5/.
 		cp -f tzselect.8 zdump.8 zic.8 $(DESTDIR)$(MANDIR)/man8/.
